common-custom-user-data-maven-extension/README.md

```diff
@@ -18,6 +18,16 @@ The additional tags, links and custom values captured by this extension include:
 See [CustomBuildScanEnhancements.java](./src/main/java/com/gradle/CustomBuildScanEnhancements.java) for details on what data is
 captured and under which conditions.
 
+#### Version compatibility
+
+This table details the version compatibility of the Common Custom User Data Maven extension with the Gradle Enterprise Maven extension.
+
+| Common Custom User Data Maven extension versions | Gradle Enterprise Maven extension versions |
+| ------------------------------------------------ | ------------------------------------------ |
+| `1.5.1`                                          | `1.10.1+`                                  |
+| `1.3` - `1.5.0`                                  | `1.6.5+`                                   |
+| `1.0` - `1.3`                                    | `1.0+`                                     |
```

Comment:
@facewindu 1.3 appears in both rows? Is that correct? it seems to me that it that the last line should be `1.0 - 1.2`.

| `1.7+`                                              | `1.10.1+`                                 |
| `1.3` - `1.6`                                    | `1.6.5+`                                  |
| `1.0` - `1.2`                                    | `1.0+`                                     |